README.md

```diff
@@ -0,0 +1,107 @@
+
+This tool reuses the functionalities of the existing [amazon_cloud_code_generator](https://github.com/ansible-collections/amazon_cloud_code_generator) and [vmware_rest_code_generator](https://github.com/ansible-collections/vmware_rest_code_generator). These generators will be archived in the future.
+
+The final goal is to integrate this tool with the [``content_builder``](https://github.com/ansible-community/ansible.content_builder)
+and extend the usage to other openAPI based cloud platforms.
+
+## Requirements
+
+The `amazon.cloud` modules generation relies on the CloudFormation client. Hence, the following requirements must be met:
+- `boto3 >= 1.20.0`
+- `botocore >= 1.23.0`
+
+Apart from the above requirements, the tool needs
+
+- `Python 3.9`
+- `tox`
+
+### Usage
+
+The ``gouttelette`` generator is capable of performing the following functionalities
+
+- [`refresh_modules`](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/cmd/refresh_modules.py) : Generates the modules specified in [resources.py](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/cmd/resources.py). It is important to make sure the rest schema and modules list files are present in the default location (details below), if not passed as input arguments.
+```
+  python -m gouttelette.cmd.refresh_modules --collection <collection_name> --target-dir <path> --modules <path> --next-version <version> --schema-dir <path>
+```
+
+| Input args | Description  |
+| ------ | ------ |
+| collection | The collection for which the modules are generated (values: amazon_cloud or vmware_rest, default: amazon_cloud) |
+| target-dir | Location of the target repository (default: ./cloud or ./vmware_rest) |
+| modules | Location of the [modules.yaml](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/config/amazon_cloud/modules.yaml) file (default: gouttelette/config/<collection>)|
+| next-version | The next major version (default: "TODO" |
+| schema-dir | Location of the collected schemas (default: ./gouttelette/api_specifications/<collection> |
+
+- [`refresh_schema`](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/cmd/refresh_schema.py): This command is for the amazon.cloud . It uses the AWS CloudFormation API to generate the rest schemas for the modules mentioned in the modules.yaml.
+```
+  python -m gouttelette.cmd.refresh_schema  --schema-dir <path>
+```
+| Input args | Description  |
+| ------ | ------ |
+| schema-dir | Location where the generated schemas will be stored (default: ./gouttelette/api_specifications/amazon_cloud |
+
+- [`refresh_examples`](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/cmd/refresh_examples.py): Generates the examples by executing the integration tests and populates the EXAMPLE block of the module documentation.
```

Comment:
I guess this is a bit unclear to me, what do you mean by "executing the integration tests "?